### PR TITLE
purge URLs on page/post update

### LINKED ISSUE #200: Automatic cache purge doesn't work when switching a page from public => private
This plugin is usually awesome and works well for purging the CF cache when a post is updated. However it doesn't do it if you change the visibility of an existing page from public to private-- the cached public version stays around.

(I imagine this bug is because it thinks "oh, I don't need to purge the cache for this private post" without taking into consideration that it used to be public a moment ago)

### PR DESCRIPTION
**Pull request by macbookandrew**
Fixes #200

### CODE REVIEW

File: cloudflare.loader.php [hunk 1]
```diff
@@ -83,6 +83,7 @@
     'deleted_post',                     // Delete a post
     'edit_post',                        // Edit a post - includes leaving comments
     'delete_attachment',                // Delete an attachment - includes re-uploading
+    'post_update',                      // Update a post
```
Comment (thellimist) [inline cloudflare.loader.php]: Could you link which hook this is?

Comment (macbookandrew) [reply cloudflare.loader.php]: Sorry…just pushed a change to fix the typo. Here’s the hook: https://developer.wordpress.org/reference/hooks/post_updated/

**Comment by macbookandrew** [comment]
@thellimist This change purges the cache for single URLs when the page/post/CPT is updated. I also updated the documentation to match the new behavior.

**Comment by thellimist** [comment]
Hi @macbookandrew,

The core developers of this project has changed. @IcyApril could you notify Patryk. 

Thanks

**Comment by knyttl** [comment]
Hey, would this also purge the cache on scheduled page publishing?

**Comment by patryk** [comment]
Thanks!

**Pull request merged**